src/encord_active/lib/encord/local_sdk.py

```diff
@@ -53,42 +56,50 @@ class LabelRowMetadata(TypedDict):
     annotation_task_status: str
 
 
+@dataclass
+class DataRowFrame:
+    path: Path
+    uid: str
+
+    def __post_init__(self):
+        self.path = self.path.resolve()
+
+
 class LocalDataRow(DataRow):
     def __init__(
         self,
         uid: str,
         label_hash: str,
         title: str,
         data_type: DataType,
-        path: Path,
+        frame: Union[DataRowFrame, List[DataRowFrame]],
```

Comment:
[curious]🤔 

Should we maybe name this differently since we are already using frame differently in other contexts (frame mostly refers to frame number).

of the top of my head is something like `media` maybe?

Reply:
media is nice because it is the same for singular and plural. Fixed it.